src/components/client/Client.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import IconArrowExpand from '../../assets/icons/icon-arrow-expand.svg';
+import styles from './Client.module.css';
+
+const Client = ({ client: { client } }) => {
```

Comment:
Having a `client` object in the `client` object seems to indicate the data structure isn't right.